vtex-apps/store-components
# Add new prop to Shipping Simulator

#### What problem is this solving?

Give a freight visibility option.
If the product has gifts attached, you can choose a property where you choose to show the shipping options for each item, or add up all values ​​by modality.

#### How to test it?

1- Enter the following workspace: 
[Workspace](https://sdtio763--diageo.myvtex.com/whisky-johnnie-walker-red-label-embalagem-comemorativa-de--200-anos--1l-759895_pai/p)
2 - Add the sumAllValues ​​property with the value "true" in the component;
3 - See that now, the freight values ​​will be shown following the respective grouping of their types;

#### Screenshots or example usage:

Before and with "sumAllValues" as false:

![PR-1](https://user-images.githubusercontent.com/80407814/118549631-214eeb00-b732-11eb-8819-1f79acf498b3.png)

![PR-2](https://user-images.githubusercontent.com/80407814/118549634-21e78180-b732-11eb-96c5-cbf76a70a356.png)

After with "sumAllValues" as true:

![PR-3](https://user-images.githubusercontent.com/80407814/118549760-480d2180-b732-11eb-8537-b94bf78c36cb.png)

![PR-4](https://user-images.githubusercontent.com/80407814/118549762-48a5b800-b732-11eb-88ed-1413f56017e4.png)
